core/src/subgraph/instance_manager.rs

```diff
@@ -347,6 +348,19 @@ where
                 Ok(true)
             }
         })
+        // Log and drop the errors from the block_stream
+        // The block stream will continue attempting to produce blocks
+        .then(move |result| match result {
+            Ok(block) => Ok(Some(block)),
+            Err(e) => {
+                error!(
+                    logger_for_block_stream_errors,
+                    "Block stream produced a non fatal error = {}", e,
```

Comment:
How about this?
```rust
debug!(
  logger_for_stream_errors,
  "Block stream produced a non-fatal error";
  "error" => format!("{}", e),
);
```

Reply:
The "non-fatal" wording sounds a bit dismissive, there could be errors that the subgraph can't recover from.

Reply:
Which errors are you referring to?

Reply:
Don't know what specific errors could happen here, but "non-fatal" sounds like it will go away by itself and is not interrupting progress, when it could be something like the DB or the network that got disconnected and will require action to fix.

Reply:
Now I get that this is meant to signal that there's nothing wrong with the subgraph itself, which makes sense.